src/lfortran/parser/semantics.h

```diff
@@ -1499,12 +1499,26 @@ void add_ws_warning(const Location &loc,
                 "Use 'end if' instead of 'endif'",
                 {loc},
                 "help: write this as 'end if'");
+        } else if (end_token == yytokentype::KW_REAL) {
+                if(a_kind == 4){
+                        diagnostics.parser_style_label(
+                        "Use real(4) x or real(4) :: x instead of real*4 x",
```

Comment:
I would phrase this as;
```suggestion
                        "Use real(4) instead of real*4",
```
and the same for the other messages below.